starport_template/lib/utils/base_env.dart

```diff
@@ -27,7 +27,7 @@ class BaseEnv {
 
 String get envLcdPort => const String.fromEnvironment('LCD_PORT', defaultValue: '1317');
 
-String get envGrpcPort => const String.fromEnvironment('GRPC_PORT', defaultValue: '9091');
+String get envGrpcPort => const String.fromEnvironment('GRPC_PORT', defaultValue: '9090');
```

Comment:
are we sure about this port change?

Reply:
Yes we are sure. I tested it locally using tests as well. It now only works on `9090`